src/index.js

```diff
@@ -1,35 +1,106 @@
 
-const todos = [{ description: 'Wash clothes', completed: false, index: 1 },
-  { description: 'Make dinner', completed: false, index: 2 },
-  { description: 'wash dishes', completed: false, index: 3 },
-  { description: 'clean house', completed: false, index: 4 }];
-
-todaysTodo.innerHTML = `<h4 class="title">Today's To do</h4>
-<button class="refreshBtn"><img src="${refresh}" alt=""></button>`;
-
-addTodo.innerHTML = `<input type="text" class="input-todo" placeholder="Add to your list...">
-<button class="addBtn"><img src="${add}" alt=""></button>`;
-
-todos.forEach((todo) => {
-  const list = document.createElement('li');
-  list.className = 'list';
-  list.innerHTML = `
-  <input id="${todo.index}" class="check" type="checkbox">
-  <span class="todo-title">${todo.description}</span>
-  <button id="${todo.index}" class="more-btn"><img src="${morevert}" alt="more-vertical-icon"></button>
-  `;
-  todoList.appendChild(list);
-});
+function render() {
+  todaysTodo.innerHTML = `<h4 class="title">Today's To do</h4>
+  <button class="refreshBtn"><img src="${refresh}" alt=""></button>`;
+  addTodo.innerHTML = `<input type="text" class="input-todo required" placeholder="Add to your list...">
+  <button class="addBtn"><img src="${add}" alt=""></button>`;
+
+  completeAll.innerHTML = `<input class="check" type="checkbox">
+  <span class="todo-title bold">Complete To Do list project</span>
+  <button class="more-btn"><img src="${morevert}" alt="more-vertical-icon"></button>`;
+}
+
+render();
+
+display();
+
+const lists = document.querySelectorAll('.list');
+const inputTodo = document.querySelector('.input-todo');
+const addBtn = document.querySelector('.addBtn');
+
+function updateTodo(e) {
+  const idUpdate = e.target.id;
+  const newValue = document.querySelector('.new-input').value;
+  updateTask(idUpdate, newValue);
+  display();
+}
 
-completeAll.innerHTML = `<input class="check" type="checkbox">
-<span class="todo-title bold">Complete To Do list project</span>
-<button class="more-btn"><img src="${morevert}" alt="more-vertical-icon"></button>`;
+function editTodo(idTask) {
+  lists.forEach((list) => {
+    if (list.childNodes[2].firstChild.id === idTask) {
+      const elem = document.createElement('div');
+      elem.className = 'edit-cont';
+      let el = document.createElement('input');
+      el.type = 'text';
+      el.className = 'new-input';
+      elem.appendChild(el);
+      el = document.createElement('button');
+      el.className = 'update';
+      el.id = idTask;
+      el.innerText = 'Update';
+      el.onclick = updateTodo;
+      elem.appendChild(el);
+      list.replaceChild(elem, list.childNodes[1]);
+      list.removeChild(list.childNodes[2]);
+    }
+  });
+}
+
+function editTask(e) {
+  const idTask = e.target.id;
+  editTodo(idTask);
+}
+
+function deleteTask(e) {
+  const removed = e.target;
+  const { id } = removed;
+  removeTask(id);
+  display();
+  window.location.reload();
+}
```

Comment:
- [ ] Great job so far 👏🏻 However, According to the project requirements. Deleting a book should update all remaining items' indexes, so they represent the current list order and are unique. Kindly address this
 - Here is what your local storage console looks like after deleting some books:
```json
0 {description: "b1", index: 1, completed: false}
1 {description: "b3", index: 3, completed: false}
2{description: "b4", index: 3, completed: false}
3{description: "b5", index: 4, completed: false}
```

![image](https://user-images.githubusercontent.com/79058364/221350630-9181b1b5-6236-4303-a38f-9a3593d5efcd.png)

- Please correct this 💯 👍🏻 

- Note: since the `b3 and b4` has the same index, this cause you app to have this kind of issue 👇🏻 Once you update b3, the b4 get updated as well. Please fix this 👍🏻 

![image](https://user-images.githubusercontent.com/79058364/221350878-41a458a6-c7c6-45e1-aad4-9a0b8f0bbced.png)